packages/ui/aurora-theme/src/config/tailwind.ts

```diff
@@ -31,7 +31,7 @@ const shadeNumbers: number[] = /* [...Array(19)].map((_, i) => 50 + i * 50); */
 ];
 
 const broadShadeNumbers: number[] = [
-  12, 25, 37, 50, 75, 100, 125, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 925,
+  5, 12, 25, 37, 50, 75, 100, 125, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 925,
```

Comment:
It looks like this token didn’t get used, recommend reverting since it’s probably too close to white anyway.

Reply:
i was looking for a bg color that was as white as possible but still had some contrast for the stack sections. adding 5 here didn't work and i couldn't figure out why.